pkg/v1/providers/infrastructure-azure/v1.4.0/yttcc/overlay.yaml

```diff
@@ -81,6 +81,11 @@ spec:
     version: #@ data.values.KUBERNETES_VERSION
     controlPlane:
       replicas: #@ data.values.CONTROL_PLANE_MACHINE_COUNT
+      #@overlay/match missing_ok=True
+      metadata:
+        annotations:
+          #! VVV TODO(tenczar) os-name handling
+          run.tanzu.vmware.com/resolve-os-image: os-name=ubuntu,os-version=2004,azure-gallery=TKG_Gallery
```

Comment:
This should be in the TKG default clusterclass rather than the overlay?

Reply:
remove azure-gallery